licenses/ATMEL_LLA_SAM-BA

```diff
@@ -1,32 +1,32 @@
+
+5. Notice and Protection.  Developer agrees not to remove or destroy any proprietary trademark or copyright markings or notices placed upon or contained within the Software or any related documentation. 
+
+6. Export.  Developer acknowledges that the certain laws and regulations may restrict the export and re-export of the Software.  Developer will not export or re-export any Software (including the diskettes, related documentation and/or any hardware peripherals) in any form without the appropriate United States and foreign governmental approval.
+
+7. Termination.  The license will automatically terminate if Developer fails to comply with any of the terms and conditions of the license.  Upon termination for any reason, Developer will immediately destroy or return to Atmel the Software, including all documentation and all whole or partial copies of the Software.
+
+8. LIMITATION OF LIABILITY.  IN NO EVENT SHALL ATMEL BE LIABLE TO DEVELOPER OR ANY THIRD PARTY FOR ANY CONSEQUENTIAL, SPECIAL, INDIRECT OR INCIDENTAL DAMAGES ARISING OUT OF OR RELATING TO THE SOFTWARE OR ANY OTHER ASPECT OF THIS AGREEMENT, EVEN IF ATMEL HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES. IN NO EVENT SHALL THE LIABILITY OF ATMEL ARISING OUT OF OR RELATING TO THIS AGREEMENT EXCEED THE GREATER OF ONE THOUSAND U.S. DOLLARS (US$1,000) OR THE PRICE PAID BY DEVELOPER TO ATMEL FOR THE SOFTWARE.
+
+9. General.  This Agreement and all transactions concluded hereunder shall be governed by the laws of the State of California, as such laws are applied to contracts entered into and performed entirely in California by California residents.  Any litigation relating to this Agreement shall be subject to the exclusive jurisdiction of the state courts located in Santa Clara County, California, or the federal courts located in the Northern District of California.  If any provision of this Agreement is held to be invalid, illegal or unenforceable, that provision shall be construed in such a manner that it becomes valid and enforceable and so as to reflect most closely the intent of the parties in agreeing upon the provision in the first place, and the remaining provisions of this Agreement shall continue in full force and effect and shall not in any way be affected or impaired by any such determination of invalidity, illegality or unenforceability.
+
+THIS AGREEMENT IS THE ENTIRE AND EXCLUSIVE AGREEMENT BETWEEN ATMEL AND DEVELOPER AND SUPERSEDES ALL PRIOR ORAL AND WRITTEN AGREEMENTS AND COMMUNICATIONS BETWEEN THE PARTIES PERTAINING TO THE SUBJECT MATTER OF THIS AGREEMENT.  NO DIFFERENT OR ADDITIONAL TERMS WILL BE ENFORCEABLE AGAINST ATMEL UNLESS ATMEL GIVES ITS EXPRESS WRITTEN CONSENT, INCLUDING AN EXPRESS WAIVER OF THE TERMS OF THIS AGREEMENT.
+
+
+Atmel Corporation
+1600 Technology Drive
+San Jose, CA 95110
+http://www.atmel.com
+
+
```

Comment:
could you please tell me what exactly changed here?

Reply:
that was unwanted whitespace changes introduced by editor, I have fixed it and uploaded new pull. Thanks for reporting it.